pom.xml

```diff
@@ -732,11 +732,11 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-compiler-plugin</artifactId>
-                    <version>3.5.1</version>
+                    <version>3.6.1</version>
                     <configuration>
                         <encoding>UTF-8</encoding>
-                        <source>1.6</source>
-                        <target>1.6</target>
+                        <source>1.7</source>
+                        <target>1.7</target>
```

Comment:
The source version should definitely be 1.7, but I do not know about the target version. From Cucumber-JVM v1.1.6 (which was quite long ago now), the intention was require java7 to built, but only java6 to use Cucumber-JVM. For instance Android supported only java6 bytecode for a long time, even thought it looks like the current Android versions and tool tests also support java7. Maybe we should stay on 1.6 as the target version.